mail-overview.adoc

```diff
@@ -34,7 +34,7 @@ Trash:: All deleted emails are moved to this folder until you _Empty the Trash_,
 Folders:: Sometimes, you may want a folder for storing specific emails.
 All such folders are listed here.
 
-Tags:: It also lists your saved searches and tags that other emails may carry.
+Tags:: It also lists your tags that other emails may carry.
```

Comment:
Can we add a short description for "Saved Searches"?

Reply:
Added.  Please see the new text in mail-search.adoc